packages/frontier-plugins/plugin-qa/webpack.config.js

```diff
@@ -10,7 +11,11 @@ module.exports = function(env = {}) {
     mode: 'production',
     entry: resolve(__dirname, 'esnext', 'index'),
     resolve: {
-      extensions: ['.js'],
+      extensions: ['.js', '.jsx', '.ts', '.tsx'],
```

Comment:
whats with the added extentions?

Reply:
was another attempt to fix the typescript aliasing.